src/main/controllers/ChallengeController.kt

```diff
@@ -17,8 +17,7 @@ class ChallengeController: DefaultController<Challenge>(), RestController<Challe
     override fun create(user: UserAccount?, requestData: RequestData): SOAResult<Challenge> {
         validateApiKey(user!!, requestData)
         return DaoService.execute {
-            val challengeNamespace = JsonHelper.parse<ChallengeNamespace>(requestData.body["challengeNamespace"]!! as String)
-
+            val challengeNamespace = JsonHelper.parse<ChallengeNamespace>(Klaxon().toJsonString(requestData.body["challengeNamespace"]))
```

Comment:
instead of initializing another Klaxon here, just us ethe one in JsonHelper

Reply:
what is in the body? its not a json string already?

Reply:
the logs indicate that its a JsonObject, not a String

Reply:
ya thats something u shouldnt change here -- change the test to pass a string

Reply:
actually it should be a Map -- if u look at RequestData how its populating body